app/controllers/psa/remove/PsaRemovalDateController.scala

```diff
@@ -62,16 +63,42 @@ class PsaRemovalDateController @Inject()(
     with I18nSupport
     with Retrievals {
 
+  //TODO RENAME SCHEME OPEN DATE VARIABLE? WRONG DATA
```

Comment:
Can you either remove this comment or make it clearer what needs to be done? Also, it should be // TODO: I think to be picked up and highlighted